index.bs

```diff
@@ -1957,6 +1977,11 @@ To <dfn>obtain a randomized source response pick rate</dfn> given a [=randomized
 1. Let |numPossibleValues| be the [=set/size=] of |possibleValues|.
 1. Return |numPossibleValues| / (|numPossibleValues| - 1 + e<sup>|epsilon|</sup>).
 
+To <dfn>obtain pick rate for a source with attribution scope</dfn> given a positve 32-bit integer |numSourceStates|, a positve 32-bit integer |numEventStates|, a positve 32-bit integer |numUsedScopes|, a positive and a double |epsilon|:
+1. Let |pickRateForSource| be (|numSourceStates| - 1) / (|numSourceStates| - 1 + e<sup>|epsilon|</sup>).
```

Comment:
`pickRateForNavigation`?

Reply:
If the source is event, why is it not counted among `numUsedScopes`?

Reply:
renamed to `pickRateForNavigation `. Done.